src/main/java/Duke.java

```diff
@@ -2,33 +2,35 @@
 
 public class Duke {
 
-    public static void printLine(){
+    public static void printLine() {
         System.out.println("\t" + "-----------------------------------------");
     }
 
-    public static void greeting(){
-        String logo = "\t" + " ____        _        \n"
-                + "\t" + "|  _ \\ _   _| | _____ \n"
-                + "\t" + "| | | | | | | |/ / _ \\\n"
-                + "\t" + "| |_| | |_| |   <  __/\n"
-                + "\t" + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("\t" + "Hello from\n" + logo);
+    public static void greeting() {
+        String logo = "\t" + "  ____        _        \n"
+                + "\t" + " |  _ \\ _   _| | _____ \n"
+                + "\t" + " | | | | | | | |/ / _ \\\n"
+                + "\t" + " | |_| | |_| |   <  __/\n"
+                + "\t" + " |____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("\t" + " Hello from\n" + logo);
         printLine();
 
-        System.out.println("\t" + "Hi! This is Duke!");
-        System.out.println("\t" + "I'm glad to be at your service.");
-        System.out.println("\t" + "What can I help you with?");
+        System.out.println("\t" + " Hi! This is Duke!");
+        System.out.println("\t" + " I'm glad to be at your service.");
+        System.out.println("\t" + " What can I help you with?");
         printLine();
     }
 
     public static void bye() {
-        System.out.println("\t" + "Bye. Hope to see you again soon!");
+        System.out.println("\t" + " Bye. Hope to see you again soon!");
         printLine();
     }
 
-    public static void echo(String task) {
+    public static void echo(Task task, TaskList taskList) {
```

Comment:
Perhaps you can rename function's name for this situation. Just by the name alone I thought it meant that it simply echos whatever input was given through CLI, but thats not the case. Try giving it a more relevant name to its feature, e.g. "printTaskAddedMessage()".